src/primitives/block.cpp

```diff
@@ -15,7 +15,7 @@ uint256 CBlockHeader::GetHash() const
     return SerializeHash(*this);
 }
 
-uint256 CBlock::BuildMerkleTree(bool* fMutated) const
+uint256 CBlock::ComputeMerkleRoot(bool* fMutated) const
```

Comment:
Unrelated to this PR,  but is there any reason we prefer the optional arguments compared to a `tuple` return value?